.github/workflows/benchmarks-nightly.yml

```diff
@@ -35,3 +35,32 @@ jobs:
         sycl_config_params: ''
         sycl_repo: 'intel/llvm'
         sycl_commit: ''
+
+  nightly3:
+    name: Compute Benchmarks Nightly level-zero
+    uses: ./.github/workflows/benchmarks-reusable.yml
+    with:
+      str_name: 'level_zero'
+      runner: 'DSS-L0-BMG'
+      preset: 'Full'
+      pr_no: 0
+      bench_script_params: '--save Baseline_BMG_L0'
+      sycl_config_params: ''
+      sycl_repo: 'intel/llvm'
+      sycl_commit: ''
+
+  nightly4:
+    # we need to wait until previous job is done so that the html report
+    # contains both runs
+    needs: nightly3
+    runner: 'DSS-L0-BMG'
```

Comment:
this runner is put in the wrong place - should be put as a param to the reusable job

Reply:
I don't get it. It's part of the runner reusable job? right here:
https://github.com/oneapi-src/unified-runtime/pull/2737/files#diff-ed216951def5b0a1b03955d3600bda8c6e7e4738f5e9279d558fa43535634804R37-R40

Reply:
I mean here. It should be under `with:`
If used as it is now it won't be passed to the reusable job

Reply:
aaa, ok, thanks :D done.